apps/electron-app/src/renderer/src/components/ui/text-input.tsx

```
@@ -4,6 +4,7 @@ interface TextInputProps {
   value: string;
   onChange: (value: string) => void;
   onEnter?: () => void;
+  onKeyDown?: (event: KeyboardEvent<HTMLTextAreaElement>) => boolean | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)